tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetCore5AsmToggle.cs

```diff
@@ -24,30 +28,37 @@ public class AspNetCore5AsmToggle : AspNetBase, IDisposable
         public AspNetCore5AsmToggle(ITestOutputHelper outputHelper)
             : base("AspNetCore5", outputHelper, "/shutdown", testName: nameof(AspNetCore5AsmToggle))
         {
+            SetEnvironmentVariable(ConfigurationKeys.Rcm.PollInterval, "500");
         }
 
         [SkippableTheory]
-        [InlineData(true)]
-        [InlineData(false)]
-        [InlineData(null)]
+        [InlineData(true, ApplyStates.ACKNOWLEDGED)]
+        [InlineData(false, ApplyStates.UNACKNOWLEDGED)]
```

Comment:
I'm a little bit confused by this. In `CheckAckState` we are asserting on the `ApplyState` for the `FEATURES` product. 
Shouldn't this be the `ASM` product? 

I assume we should always be applying and acknowledging the `FEATURES` product, regardless of whether ASM is enabled, as that might contains flags that are related to other products as well?

Reply:
I think what we're doing is okay because:
- The FEATURES product contains the on / off switch for app sec, ASM product contains to activate or deactivate individual rules, they are two separate features that need a separate ack.
- You don't ack a product but a file within a product.

We should really include the path as part of the assert, but I don't we've settled on a path name yet. I think the FEATURES product needs quite careful evolution to ensure it suites everyone's needs. I'll start a thread internally.